Source/Core/VideoBackends/Software/Rasterizer.cpp

```diff
@@ -320,11 +320,11 @@ void DrawTriangleFrontFace(const OutputVertexData* v0, const OutputVertexData* v
     scissorTop = 0;
 
   s32 scissorRight = bpmem.scissorBR.x - xoff - 341;
-  if (scissorRight > EFB_WIDTH)
+  if (scissorRight > (s32)EFB_WIDTH)
```

Comment:
We prefer either `static_cast` or `s32(thing)` forms of casting over C-style casting.

Reply:
okay